<launch>
    <arg name="namespace" default="cgras" doc="The namespace for the project" />
    <arg name="use_gui" default="false" />
    <arg name="use_rviz" default="true" />
    <arg name="sim_robot" default="false" /> 
    <arg name="default" default="true" />  
    <arg name="objects_config" default="$(find cgras_robot)/config/objects.yaml" />  
    <arg name="workarea_config" default="$(find cgras_robot)/config/workarea.yaml" />    

    <!-- Launch the (simulated) robot node -->
    <group if="$(arg default)">
        <node pkg="cgras_robot" type="robot_server_agent.py" name="robot_server_agent" output="screen">
        </node>
        <node pkg="cgras_robot" type="cgras_robot_server_agent.py" name="cgras_robot_server_agent" output="screen">
        </node>   
    </group>

    <group if="$(arg sim_robot)">
        <include file="$(find cgras_scene_config)/launch/demo.launch">
            <arg name="use_gui" default="$(arg use_gui)" />
            <arg name="use_rviz" default="$(arg use_rviz)" />
        </include>
    </group>


</launch>   